src/infinite-hits/infinite-hits.ts

```diff
@@ -32,7 +32,7 @@ import { noop } from '../utils';
       </div>
 
       <button
-        class="{{cx('loadMore')}} {{this.state.isLastPage ? cx('loadMore', 'disabled') : ''}}"
+        [ngClass]="[cx('loadMore'), this.state.isLastPage ? cx('loadMore', 'disabled') : '']"
```

Comment:
You can use the form:

```js
cx('loadMore', { 'disabled': this.state.isLastPage })
```

See: https://github.com/JedWatson/classnames#usage

Reply:
Nope it's not this package it's a custom [implementation](https://github.com/algolia/angular-instantsearch/blob/f83117d925bef4dd96d0bafafd1e89e3b56f5cea/src/utils.ts#L1-L25).

Reply:
Oh, right...